pkg/proto/datadog/api/README.md

```diff
@@ -0,0 +1,56 @@
+## gRPC: Protobuf and Gateway code generation 
```

Comment:
I can't seem to add a comment about it since it's just a rename but [`pkg/proto/datadog/README.md`](https://github.com/DataDog/datadog-agent/blob/c5ce993c5ec133f57584302d2d35009477826bfa/pkg/proto/datadog/README.md) has some outdated info on it now that it moved.